src/components/Tool/FixMailingAddresses/FixMailingAddresses.tsx

```diff
@@ -396,7 +395,7 @@ const FixMailingAddresses: React.FC<Props> = ({
                           />
                           {t('Confirm {{amount}} as {{source}}', {
                             amount: totalContacts,
-                            source: defaultSource,
+                            source: sourceToStr(t, defaultSource),
```

Comment:
You use in 2 places, it could be made into a variable to make it easier to maintain.